builtin/providers/aws/resource_aws_sns_topic_subscription.go

```diff
@@ -55,16 +69,17 @@ func resourceAwsSnsTopicSubscription() *schema.Resource {
 func resourceAwsSnsTopicSubscriptionCreate(d *schema.ResourceData, meta interface{}) error {
 	snsconn := meta.(*AWSClient).snsconn
 
-	if d.Get("protocol") == "email" {
-		return fmt.Errorf("Email endpoints are not supported!")
-	}
-
 	output, err := subscribeToSNSTopic(d, snsconn)
 
 	if err != nil {
 		return err
 	}
 
+	if output.SubscriptionArn != nil && *output.SubscriptionArn == "pending configuration" {
```

Comment:
I think this a typo `s/configuration/confirmation/` - maybe pull the string out into a constant?